controllers/pattern_controller.go

```diff
@@ -331,6 +349,11 @@ func (r *PatternReconciler) applyDefaults(input *api.Pattern) (error, *api.Patte
 		output.Spec.ClusterGroupName = "default"
 	}
 
+	// interval cannot be less than 30 seconds to avoid drowning the API server in requests
+	if output.Spec.GitConfig.PollInterval < 30 {
```

Comment:
I'd suggest at least an hour.

Reply:
It's set to [180 seconds](https://github.com/hybrid-cloud-patterns/patterns-operator/pull/44/files#diff-ee09283b7b2366c378d550052129868e73c3ade74c8dd66a9123d0801439947eR380), to match ArgoCD's interval. I think you're looking at an old commit because I see 180 if I filter through changes from all commits.